qdb/tracer.py

```diff
@@ -123,6 +153,25 @@ def _init(self, config=None, merge=False, **kwargs):
                 RemoteOutput(self.cmd_manager, '<stderr>'),
             )
 
+    def bound_cmd_manager():
+        def fget(self):
+            return self.__cmd_manager
+
+        class BoundCmdMangaer(object):
```

Comment:
Is there a reason this class needs to be defined inline here?  I don't think we're closing over anything.